TheWarWithin/RogueAssassination.lua

```diff
@@ -2954,4 +2955,4 @@ spec:RegisterSetting( "allow_shadowmeld", nil, {
CyGkMnvYfJAFw4beRpoZZlooRMN98lMm6SNtF)g1CwyNQC9p36qKR)iJZmUW5LNI2EU29lGJ0qPGUDCf6hhiJWKUc3ClI2h9IoUCeD2R1QQgXJp6XuWKXcxN)Q3gWDftBGFwAG980i2xCwpIgWUN9(ghq0219APLRPEobtHrD3)DSSnXJwDLv1qacyEGFxg(Dv9p8zqumv99kqnFNnkZjnH9obXIxjaPrIi2ZUheOtsNxHj0a16P7hwMs5xsZpBJNQLaxZ((6f6En7dAWw0qpE4u7cOVoBUBuEwl3cGOsZRUdNTNaLqddLm9gst1vVXyp)LdAid3VoSA7UN8IjNnKBSPbhYb5qg2Ytx)fBtf06CxKoCardh3RprpcFXKtHTGDh8c88EKV2lPXCfvKOvIF2lhWdRVlnkC3KtKnyr3gnVZbz0hJ2gUNd)mMiKQZ2)n0Bdaot2nX1Sp9g8TNhXZrdfjNW2FWE0TZRwiDntstV(unXRkmdhnW7fGfGhF02HObc4s1SU96t54mQNs8QeySknl7RVvb1ydcxbbEHWLcU9NMhfBWmyV(A3U5yKJ90ZuutlNhPyvC3f26VCajHtX2YmNg3fg9oId9p959f(i1ODCE2zdEETaKvMEa9)bGeGJqxOlLJY3IMNS(bczHD9RELAAQW5(Morr(nsGmGZ9ajNllVx(AAHcBIm)6HHf(LUPEqRvZnmsKTRcfVKHk7RVmtOqgPwPXrPLhi2oPLEGlX0oO)(0(fObf2l11jW(THd2)ahzrIRJLP9fMbQQ6Ke13DonNmwPlt80Zr2I)mPJVUBgWzdaJi(So9QVeRt0gN(RcdXj7OMrzNExVcZVXj(QRl)K(LmIpnDRcW0rT8hFu5mksmzFRG0ZRhUNaI6nFNNnU93YnXDWfN6ZRYA0MNsiaRnUeOouEXubsu6xlvtEYVvQ0sS1S67llHNW)f)MMIGQkt4TYP6g(iVf7E)e6zmJ8ULxtu(r3TlIaJ7SUUvnh1fXebCy6hv9f8Gk36eQYJoYHNH0KDBsQWBq3x9rDcmnVlIKnsbTRPuPgRVB)vvskoUfOoFY4b0g5g1RjY2axKxvhg7a1eLiEt(T8sRVhqC3x3PseaHH11gWchg6DqJwDPNVkVcgPpQV72uQNJgmAGC3xY7oPEhqx6pAceRa9FnV4J6cPvZ(EJlePZ9qpbP7gmaYSWZ)qlDdn)x0yPpbSpEU3(kDlf1D9jbcZ9(v)AesVgySuYpWTBHKxprc5v95Kte2XReiFwx85VB3yRIsMSZ3HqBJTpPTlZTxn)onRE(y57VX2c0F)nPvRE79aCNQUZh7ipYoYQ(4MWgIv4Vlklj53rAhQ9DffNf1NuCQN5NwN(6rUsZF7auZVpQAcnkwenPTT1WX3Iz791P53JygLKGOobNmS3bBb)9eZQgjHIzzboqrqHQ42M4j5sA5ZagyeZkbmZltUbV8cWamG5pI1ZRquDrWwcjYFv1piDD81cjYTpZFyn6)a8HL5vcCuoRKpSXueNd4Ejs9dbFjUbfXtky9aSQBvCoc9jJO8m(z9U)kEBVlL1jnMT3mJl1xdlVT99YMByV4qwtdl711rM1tSr7XIHPtuKyxfteBAlpt2yJMhwrhyLhSNJnojRrThD5XdPNPbd07YMgW2GotYg3gjB8(KKnYO1J)RgjJh5wcOQTCj8ISWf0AgfEbdRPSYQ(XYAcBZyz8USCLTeIBbXSOssI1jdjCJseb(VhA2CbGd28Quw3xhiChRkeir9k1Gog4emVk)bMsm1Is9wWU4X5iLBBER5GZqQci21xY8RAFMezJ(FPhLyqZMSSBV2w4gYRLcg8UhGQWn4Y3WFn8dB21lMaCpIGwOINVllLcYBSOnij8TSQB6G602AaZAynZ2CyMQxbEBo)QiG1C884DAaJYMNqS8nV9cCTU(dg7ugbYQMsgTQIGrXIW3zBGzbKsXnwRQDFmttLXKeD5LfSsqS7Tvo3(A0Ug54D3Lxyo8WyYK5bYIeO6hTMEC5HInOn(2ENI4Vr2gnWY5GcnbqLuNmC4z1n(0Et6ZLr5AqINdznRZcznEuTTjIvRhEW2dC7FUpMpDw7(WvBWWW5T2dRBFCHmKnDJO)AET0GPmDcG9IiBtMYN9C3LCGNafvMcQVCXW0RkBbnSkNlWAdnhypSZNW850ltD(qm8OW8gldSgklBxJh2RVG2Qc0X4XtgkcxWm1XOG(aroHXw3RGvXwCBCXbDJr2XBioIC8ik618KuMyglz4KiNL4U22F0gLA2lvNpYEFbgqRY7bcXRuz1MvTnC81GDDsTObrTIcSlnTanIyaY1r0A6ZQnlwgwNwJnrwPttcGep)iREAGKPOGA8c)GXTWA6dE6tFkOami0UAEHv5SUQZwTFpsbuTzGKc8tg1mhtKfRO50iTNkuVZebNfThRGQQuTWwSfXStLuXXSv3xVeOhBGD1Rf7v3irYe9MQOLl)jXziqv)6Vtzjd3i734aP0BkeN0glz9l2kAUNMuX95COU6ZJGa3TRMDlDlrpntJI57C(QlKihmOr1J81uaJ9JyTGiUkfpIynGcVQvUFOkoohvlBYONpwDAJiUMjXnj13M06CpGbbYpURxcr6jfcSfRKsqVVPFq8A66z7vZk6F6zIip5lkATFo73yl8pcC75vSscZQeuZvrRPwz0mS8)xJn8sWYQOBFa0WtyPlRV1tXphKqtwwilu9zclrwJzrC4S6XmOb0wSgZIGgLrATsRN1v51VvTjk8QK8CGO8IJw3q6ordHuLtWY4)yTgTw8G2cX6ptZk3ah5NXtE5o1LiR3Qy(9NoyFjDXKPW6AEhJdjT7jLM3NCN)Szo(OOJ916oWwjGHRWKzbX3Xcsygi(NKEbedOjTufwcJQ2W7KUq0AG)Ue113guVPOmrFo8FczQxAC0iAmOAPuF4jHPqcfEzKAaXdeUc5nla6kt8NHy3k1p(seKNNEPJDr1LqM4e87RxaHU6NhF2Byxx0x9))d]] )
```

Comment:
Is there a .simc file update or is this a missed import from a previous .simc file update?

Reply:
This should match the simc that's associated with this pr, so if there's no diff then the pack string was wrong